### PR TITLE
ImpressPages CMS config

### PR DESCRIPTION
**Pull request by rizhenkov**
It's a recommended official config from ImpressPages developers adapted to elasticweb.

### CODE REVIEW

**Comment by elasticweb** [comment]
Hi

Thanks for nginx config, please, adapt the config to a common sight, example: https://github.com/elasticweb/nginx-configs/blob/master/configs/asgard-cms.conf

**Comment by rizhenkov** [comment]
Okay, done!

File: configs/impresspages.conf [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    index index.php;
+    try_files $uri $uri/ /index.php?$args;
+}
+
+location ~ \.php$ {
+	fastcgi_split_path_info ^(.+\.php)(/.+)$;
+	include fastcgi_params;
+	fastcgi_param SCRIPT_FILENAME $request_filename;
+	fastcgi_intercept_errors on;
+	fastcgi_pass unix:/var/run/php5-{domain}.sock;
```
Comment (elasticweb) [inline configs/impresspages.conf]: Hi

Please, fix it

Comment (rizhenkov) [reply configs/impresspages.conf]: Sorry, I missed that part. Now it's may be ok.

**Comment by elasticweb** [comment]
Hi

Thanks!

**Pull request merged**